File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraGauges.Core;


namespace WindowsApplication3 {
    public partial class Main: XtraForm {
        public Main() {
            InitializeComponent();
        }

        string selectedGauge = string.Empty, selectedGaugeContainer = string.Empty;

        private void OnRunStyleChooser(object sender, EventArgs e)
        {
            switch (selectedGauge)
            {
                case "Circular Gauge":
                    StyleChooser.Show(circularGauge1);
                    break;
                case "Linear Gauge":
                    StyleChooser.Show(linearGauge1);
                    break;
            }
        }

        private void OnRunStyleManager(object sender, EventArgs e)
        {
            switch (selectedGaugeContainer)
            {
                case "Gauge1":
                    StyleManager.Show(gaugeControl1);
                    break;
                case "Gauge2":
                    StyleManager.Show(gaugeControl2);
                    break;
            }
        }

        private void OnGaugeChanged(object sender, EventArgs e)
        {
            ComboBoxEdit edit = sender as ComboBoxEdit;
            selectedGauge = edit.Text;
        }

        private void OnGaugeContainerChanged(object sender, EventArgs e)
        {
            ComboBoxEdit edit = sender as ComboBoxEdit;
            selectedGaugeContainer = edit.Text;
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraGauges.Core


Namespace WindowsApplication3
	Partial Public Class Main
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private selectedGauge As String = String.Empty, selectedGaugeContainer As String = String.Empty

		Private Sub OnRunStyleChooser(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Select Case selectedGauge
				Case "Circular Gauge"
					StyleChooser.Show(circularGauge1)
				Case "Linear Gauge"
					StyleChooser.Show(linearGauge1)
			End Select
		End Sub

		Private Sub OnRunStyleManager(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton2.Click
			Select Case selectedGaugeContainer
				Case "Gauge1"
					StyleManager.Show(gaugeControl1)
				Case "Gauge2"
					StyleManager.Show(gaugeControl2)
			End Select
		End Sub

		Private Sub OnGaugeChanged(ByVal sender As Object, ByVal e As EventArgs) Handles comboBoxEdit1.EditValueChanged
			Dim edit As ComboBoxEdit = TryCast(sender, ComboBoxEdit)
			selectedGauge = edit.Text
		End Sub

		Private Sub OnGaugeContainerChanged(ByVal sender As Object, ByVal e As EventArgs) Handles comboBoxEdit2.EditValueChanged
			Dim edit As ComboBoxEdit = TryCast(sender, ComboBoxEdit)
			selectedGaugeContainer = edit.Text
		End Sub
	End Class
End Namespace
